config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -168,3 +168,10 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=120m
       - --use-logexporter
+      resources:
+        requests:
+          cpu: 2
+          memory: 6Gi
```

Comment:
@tosi3k - you were running some experiments here. I think the outcome was that 2 CPUs were more than enough for 100-node tests, right?

Reply:
> @tosi3k - you were running some experiments here. I think the outcome was that 2 CPUs were more than enough for 100-node tests, right?

did you run the experiments with requests and limits, or just requests?

Reply:
Just the requests w/o limits and now I see it was wrong... my bad.

Reply:
Like I said in description, mine was a visual guess.  I think the actual peak I saw was somewhere around 1200m-1500m but I wanted to stick to whole numbers of CPUs for this first pass.  If you want to open a PR against https://github.com/kubernetes/k8s.io/blob/8e267aa1ed12b26897dbf9bb2d6c1294cdd76dbd/groups/groups.yaml#L679-L688 to add yourself to the group and see what I saw I'm happy to merge.